shared/js/sidebar.jsx

```diff
@@ -0,0 +1,179 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Also, I don't see why this is in shared/ I think it should be for the add-on only.